src/Microsoft.Azure.WebJobs.Host/Dispatch/DummyDispatchQueueHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host.Dispatch
+{
+    // this class is used for local testing
+    // an extension developer can test DispatchQueueHandler without 
+    // providing an azure storage account
+    internal class DummyDispatchQueueHandler : IDispatchQueueHandler
```

Comment:
>DummyDispatchQueueHandler [](start = 19, length = 25)

This can actually be used in production. For example, the AFR will use it (since they don't have real blob storage). Can we rename  this, like "InMemoryDispatchQueueHandler"